components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/EndpointCertificatesApiServiceImpl.java

```diff
@@ -71,7 +71,7 @@ public Response getEndpointCertificateContentByAlias(String alias, MessageContex
                 RestApiUtil.handleResourceNotFoundError(message, log);
             }
 
-            Object certificate = apiProvider.getCertificateContent(alias);
+            Object certificate = apiProvider.getCertificateContent(RestApiCommonUtil.getLoggedInUsername(), alias);
```

Comment:
shall we pass tenant domain instead of user name?

Reply:
+1

Reply:
Fixed